WowPacketParser/Enums/UpdateFields.cs

```diff
@@ -792,16 +792,19 @@ public enum CorpseDynamicField
 
     public enum AreaTriggerField
```

Comment:
I need it in Store/Objects/SpellAreaTrigger.cs, how can i get the enum from the 7.x module in this file ?

Not very comfortable with C# yet, "using WowPacketParserModule.V7_1_5_23360.Enums" don't seem to work